Error when using tap-notify with AVA

Whenever using tap-notify with [ava](https://github.com/sindresorhus/ava) I get this error:

``` javascript
// ava test/unit/*spec.js --tap | tap-notify

tap-notify/distributions/index.js:43
var passedOptions = Object.assign({}, DEFAULT_PASSED_OPTIONS, passed);

TypeError: undefined is not a function
    at createReporter (/Users/fernandofleury/n/lib/node_modules/tap-notify/distributions/index.js:43:30)
    at Object.<anonymous> (/Users/fernandofleury/n/lib/node_modules/tap-notify/distributions/cli.js:10:39)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

events.js:85
      throw er; // Unhandled 'error' event

Error: write EPIPE
    at exports._errnoException (util.js:746:11)
    at WriteWrap.afterWrite (net.js:777:14)

npm ERR! Test failed.  See above for more details.
```

Others tap such as tap-nyan works just fine. Am I missing something here?

--- a/.babelrc
+++ b/.babelrc
@@ -1,0 +1,4 @@
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports", "transform-object-assign"]
+}

--- a/distributions/cli.js
+++ b/distributions/cli.js
@@ -1,13 +1,13 @@
 #!/usr/bin/env node
 'use strict';
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }
-
 var _index = require('./index');
 
 var _index2 = _interopRequireDefault(_index);
 
-var reporter = (0, _index2['default'])();
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+var reporter = (0, _index2.default)();
 
 process.stdin.pipe(reporter).pipe(process.stdout);
 

--- a/distributions/index.js
+++ b/distributions/index.js
@@ -1,18 +1,18 @@
 'use strict';
 
-Object.defineProperty(exports, '__esModule', {
+var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
+
+Object.defineProperty(exports, "__esModule", {
   value: true
 });
-
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }
 
 var _nodeNotifier = require('node-notifier');
 
 var _nodeNotifier2 = _interopRequireDefault(_nodeNotifier);
 
-var _through2 = require('through2');
+var _through = require('through2');
 
-var _through22 = _interopRequireDefault(_through2);
+var _through2 = _interopRequireDefault(_through);
 
 var _tapParser = require('tap-parser');
 
@@ -22,15 +22,17 @@ var _path = require('path');
 
 var _path2 = _interopRequireDefault(_path);
 
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
 var DEFAULT_PASSED_OPTIONS = {
   title: 'Test passed.',
-  icon: _path2['default'].resolve(__dirname, '../passed.png'),
+  icon: _path2.default.resolve(__dirname, '../passed.png'),
   sound: false
 };
 
 var DEFAULT_FAILED_OPTIONS = {
   title: 'Test failed!',
-  icon: _path2['default'].resolve(__dirname, '../failed.png'),
+  icon: _path2.default.resolve(__dirname, '../failed.png'),
   sound: 'Basso'
 };
 
@@ -40,10 +42,10 @@ var createReporter = function createReporter() {
   var passed = _ref.passed;
   var failed = _ref.failed;
 
-  var passedOptions = Object.assign({}, DEFAULT_PASSED_OPTIONS, passed);
-  var failedOptions = Object.assign({}, DEFAULT_FAILED_OPTIONS, failed);
-  var p = (0, _tapParser2['default'])();
-  var stream = (0, _through22['default'])(function (chunk, enc, next) {
+  var passedOptions = _extends({}, DEFAULT_PASSED_OPTIONS, passed);
+  var failedOptions = _extends({}, DEFAULT_FAILED_OPTIONS, failed);
+  var p = (0, _tapParser2.default)();
+  var stream = (0, _through2.default)(function (chunk, enc, next) {
     this.push(chunk);
     next();
   });
@@ -59,11 +61,11 @@ var createReporter = function createReporter() {
 
   p.on('complete', function (result) {
     if (result.ok) {
-      _nodeNotifier2['default'].notify(Object.assign({
+      _nodeNotifier2.default.notify(_extends({
         message: result.pass + ' of ' + result.count + ' tests passed.'
       }, passedOptions));
     } else {
-      _nodeNotifier2['default'].notify(Object.assign({
+      _nodeNotifier2.default.notify(_extends({
         message: (result.fail || 0) + ' of ' + result.count + ' tests failed' + (errorOccuredAt ? ' at ' + errorOccuredAt : '')
       }, failedOptions));
     }
@@ -72,5 +74,5 @@ var createReporter = function createReporter() {
   return stream;
 };
 
-exports['default'] = createReporter;
+exports.default = createReporter;
 module.exports = exports['default'];

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "tap-notify",
-  "version": "0.0.3",
+  "version": "1.0.0",
   "description": "the Notifier for OS X, Linux and Windows as TAP reporter",
   "main": "./distributions/index.js",
   "bin": {
@@ -18,7 +18,10 @@
     "through2": "^2.0.0"
   },
   "devDependencies": {
-    "babel": "^5.8.34"
+    "babel-cli": "6.4.0",
+    "babel-plugin-add-module-exports": "0.1.2",
+    "babel-plugin-transform-object-assign": "6.3.13",
+    "babel-preset-es2015": "6.3.13"
   },
   "repository": {
     "type": "git",

